diff -Nur orc-orc-0.4.28/orc/meson.build orc-orc-0.4.28-dllexport/orc/meson.build
--- orc-orc-0.4.28/orc/meson.build	2017-11-20 07:15:58.000000000 -0800
+++ orc-orc-0.4.28-dllexport/orc/meson.build	2018-11-30 04:42:29.934193658 -0800
@@ -103,7 +103,7 @@
 if cc.get_id() != 'msvc'
   link_args = ['-Wl,--version-script,' + meson.current_source_dir() + '/orc.map']
 else
-  link_args = []
+  link_args = ['/DEF:' + meson.current_source_dir() + '/orc.def']
 endif
 
 # FIXME: what about shared and/or static lib? look at options?
