diff -Nur orc-orc-0.4.28/orc/meson.build orc-orc-0.4.28-windows-dllexport/orc/meson.build
--- orc-orc-0.4.28/orc/meson.build	2017-11-20 07:15:58.000000000 -0800
+++ orc-orc-0.4.28-windows-dllexport/orc/meson.build	2018-11-30 02:36:55.204322442 -0800
@@ -97,6 +97,9 @@
 endif
 
 orc_c_args = ['-DORC_ENABLE_UNSTABLE_API', '-D_GNU_SOURCE']
+if build_machine.system() == 'windows'
+  orc_c_args += ['-DORC_EXPORTS']
+endif
 
 orc_dependencies = [libm, librt, liblog]
 
